ci-setups/php-github.md

```diff
@@ -0,0 +1,314 @@
+# Continuous Integration (CI) Setup for PHP projects in Github
```

Comment:
Github  --> GitHub

Fix it elsewhere please.
Ensure it is handled for GitLab too.

Reply:
All occurrences of Github in the document have been updated to GitHub.

Similar change has been made to occurrences of Gitlab -> GitLab.